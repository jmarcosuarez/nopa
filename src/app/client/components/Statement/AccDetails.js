import React from 'react';
import PropTypes from 'prop-types';
import styles from './Styles.css';

const AccDetails = props =>
  <div className={styles.AccDetails}>
    <h1>{props.choosenBank}</h1>
    <h1>{props.surname}</h1>
    <h2>Current Account: {props.account}</h2>
  </div>;

AccDetails.propTypes = {
  // onFetchTransactions: PropTypes.func.isRequired,
  choosenBank: PropTypes.string,
  surname: PropTypes.string,
  account: PropTypes.string,
};

AccDetails.defaultProps = {
  choosenBank: '',
  surname: '',
  account: '',
};

export default AccDetails;
