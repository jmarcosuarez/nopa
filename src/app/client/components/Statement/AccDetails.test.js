/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AccDetails from './AccDetails';

test('should render correctly', () => {
  const fc = jest.fn();
  const object = {
    surname: 'Denis', 
    choosenBank: 'Santander',
    account: '09434534343434',
  };
  const wrapper = shallow(
    <AccDetails
      surname={object.surname}
      choosenBank={object.choosenBank}
      account={object.account}
    />,
  );
  expect(toJson(wrapper)).toMatchSnapshot();
});
