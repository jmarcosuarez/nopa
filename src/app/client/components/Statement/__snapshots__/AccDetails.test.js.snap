// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="AccDetails"
>
  <h1>
    Santander
  </h1>
  <h1>
    Denis
  </h1>
  <h2>
    Current Account: 
    09434534343434
  </h2>
</div>
`;
